kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/ExtensibleBot.kt

```diff
@@ -126,6 +127,17 @@ public open class ExtensibleBot(
         }
     }
 
+    /** Stop the bot. **/
+    public open suspend fun stop() {
+        getKoin().get<Kord>().shutdown()
+    }
+
+    /** Stop the bot and its [Kord] instance. Restarting the bot after closing will result in undefined behavior. **/
```

Comment:
This could be a bit more descriptive - it should explain _why_ undefined behaviour may occur (explain about the Koin closure).

That said, I think actually what'll happen is it'll just crash immediately, since the Koin context has been destroyed.